server/service/osquery.go

```diff
@@ -1606,6 +1611,134 @@ func (svc *Service) registerFlippedPolicies(ctx context.Context, hostID uint, ho
 	return nil
 }
 
+func (svc *Service) processSoftwareForNewlyFailingPolicies(
```

Comment:
Can you add a test for this method?

Reply:
This is being tested in the added test in `integration_enterprise_test.go`.
(I actually got this method 100% covered with such tests.)

Reply:
re "100%", except errors in datastore methods :)